packages/okta-vue/src/Auth.js

```diff
@@ -73,6 +73,7 @@ function install (Vue, options) {
       }
     }
   }
+  Vue.prototype.$auth.onAuthenticationError = authConfig.onAuthenticationError
```

Comment:
`onAuthenticationError` is a bit generic, as this handler is specifically for the OAuth redirect.

It might make sense to either:
1. Make this a global error handler, so a developer can inject this into any supported error callback
2. Rename this to `onOAuthRedirectError`

Reply:
`onOAuthRedirectError` sounds good 👍